### PR TITLE
Implement ohb command

### PR DESCRIPTION
**Pull request by ghost**
i haxed it

### CODE REVIEW

File: bot_modules/irc_commands.js [hunk 1]
```diff
@@ -389,6 +395,27 @@ module.exports = {
...
 	},
 
+	ohb: function(info, words) {
+		var no_ohb_message = 'We ALL love OHBs, but none is currently runningz :)))))))';
+		var req = 'battle/current';
+		req += '?filters=type~3'; // OHB only
+		var p = botb_api.request(req);
+		p.then(function(data) {
+			// TODO remove me when botb_api learns to RESPECT (?filters=type~3)
+			data = data.filter(function(battle) { return battle.type === 3; });
```
Comment (ghost) [inline bot_modules/irc_commands.js:402]: maybe you'd rather fix this on the API side?

**Comment by langel** [comment]
This isn't correct.  OHB command shouldn't give the same kind of info about a battle as the battle command.  Battle command could call the OHB display on type 3 battles, but not the other way around.

Depending on the OHB period --
warmup period: 'OHB {title} starts in ' 'format is '
entry period: 'OHB {title} entry period ends in ' 'format is '
vote period: 'OHB {title} needs your vorts!'
tally period (i think that's what its called): 'OHB {title} resluts are coming!'

**Comment by ghost** [comment]
gotcha. I guess I'll implement it in a new PR? sorry this got merged prematurely...

**Comment by langel** [comment]
Not your fault it got merged!
Also, displaying the time left until it's over isn't necessary as the end time for OHB is arbitrarily set to 24 hours after the entry period and updated when the host closes the battle.

**Pull request merged**